assets/files/etc/kubernetes/manifests/coredns.yaml

```
@@ -0,0 +1,84 @@
+---
+kind: Pod
+apiVersion: v1
+metadata:
+  name: coredns
+  namespace: kni-infra
+  creationTimestamp:
+  deletionGracePeriodSeconds: 65
+  labels:
+    app: kni-infra-mdns
+spec:
+  volumes:
+  - name: resource-dir
+    hostPath:
+      path: "/etc/kubernetes/static-pod-resources/coredns"
+  - name: kubeconfig
+    hostPath:
+      path: "/etc/kubernetes/kubeconfig"
+  - name: oc
+    hostPath:
+      path: "/bin/oc"
+  - name: clusterinfo
+    hostPath:
+      path: "/usr/local/bin/clusterinfo"
+  - name: conf-dir
+    empty-dir: {}
+  initContainers:
+  - name: render-corefile
+    image: quay.io/openshift/origin-node:latest
```

```suggestion
image: registry.svc.ci.openshift.org/openshift/origin-v4.0:base
```
This has /usr/libexec/platform-python (also python2) and dig